src/locale/catalogue.tsx

```diff
@@ -51,13 +51,13 @@ const CataloguePageTranslation = {
       colour: "Цвет",
       brand: "Бренд",
       colourType: {
-        white: "Белый",
-        cream: "Kремовый",
-        yellow: "Желтый",
-        orange: "апельсиновый",
-        red: "Красный",
-        purple: "Розовый",
-        bicolor: "Двухцветный",
+        White: "Белый",
+        Cream: "Kремовый",
+        Yellow: "Желтый",
+        Orange: "апельсиновый",
```

Comment:
Capital

Reply:
it should be that way, as color names in our catalogueFlowers array are capitalized